coalib/processes/Processing.py

```diff
@@ -382,7 +382,6 @@ def instantiate_processes(section,
     """
     filename_list = collect_files(
         glob_list(section.get('files', '')),
-        None,
```

Comment:
you can drop this.  A newcomer will do it.
https://github.com/coala/coala/issues/5341